modules/azure/resourcegroup_test.go

```diff
@@ -0,0 +1,20 @@
+package azure
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+/*
+The below tests are currently stubbed out, with the expectation that they will throw errors.
+If/when methods to create and delete resource groups are added, these tests can be extended.
+*/
+
+func TestResourceGroupExists(t *testing.T) {
+	t.Parallel()
+
+	resourceGroupName := "fakeResourceGroupName"
+	_, err := ResourceGroupExistsE(resourceGroupName, "")
+	require.Error(t, err)
```

Comment:
Should this actually check if it returns `false` without error? Since we are calling it on a resource group that we don't expect to exist?

Reply:
This pattern currently used in previously approved Exists unit test functions, i.e., getting error when subscription and/or resource group no available.